rust/chains/hyperlane-cosmos/src/trait_builder.rs

```diff
@@ -11,6 +16,46 @@ pub struct ConnectionConf {
     prefix: String,
     /// Canoncial Assets Denom
     canonical_asset: String,
+    /// The minimum gas price set by the cosmos-sdk validator
+    minimum_gas_price: RawCosmosAmount,
```

Comment:
I think saying "minimum" here may imply that the agent would maybe use a higher gas price and that this value isn't exactly what will be used for txs - I wonder if we should call this `gas_price` but still call out in the documentation that this is likely to just be the minimum gas price set by the node we're interactive with